index.bs

```diff
@@ -3365,7 +3365,7 @@ This attestation statement format is used with FIDO U2F authenticators using the
 :: fido-u2f
 
 : Attestation types supported
-:: [=Basic Attestation=], [=Self Attestation=], [=Attestation CA=]
+:: [=Basic=], [=AttCA=]
```

Comment:
Is `AttCA` a typo? (I see other references to `[=Attestation CA=]` in the file, but none to `AttCA`.)

Reply:
Oh, perhaps it works in light of [this change](https://github.com/w3c/webauthn/pull/780/commits/a9f806ce651edddefb1eec3c8d539752c9b001eb).

Reply:
yeah that's why i suggest merging pr #746 before this one, in the OP https://github.com/w3c/webauthn/pull/781#issue-294940252